addons/mail/static/src/components/composer/composer.xml

```diff
@@ -147,7 +162,7 @@
                 </button>
             </t>
             <t t-if="!env.messaging.device.isMobile and props.hasDiscardButton">
-                <button class="o_Composer_actionButton o_Composer_button o_Composer_buttonDiscard btn btn-secondary" t-att-class="{ 'o-composer-is-compact': props.isCompact }" type="button" t-on-click="_onClickDiscard">
+                <button class="o_Composer_actionButton o-last o_Composer_button o_Composer_buttonDiscard btn btn-secondary" t-att-class="{ 'o-composer-is-compact': props.isCompact, 'o-has-current-partner-avatar': props.hasCurrentPartnerAvatar }" type="button" t-on-click="_onClickDiscard">
```

Comment:
There is a layout issue with subject when replying to a mail channel message from inbox:

![image](https://user-images.githubusercontent.com/38213504/86925870-adb37900-c131-11ea-84f4-9d3c8e9a6f88.png)

Reply:
task-2293687